openeuler/testdata/cvrf-openEuler-SA-2021-1033.xml

```diff
@@ -0,0 +1,128 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<cvrfdoc xmlns="http://www.icasi.org/CVRF/schema/cvrf/1.1" xmlns:cvrf="http://www.icasi.org/CVRF/schema/cvrf/1.1">
+	<DocumentTitle xml:lang="en">An update for sane-backends is now available for openEuler-20.03-LTS and openEuler-20.03-LTS-SP1</DocumentTitle>
+	<DocumentType>Security Advisory</DocumentType>
+	<DocumentPublisher Type="Vendor">
+		<ContactDetails>openeuler-security@openeuler.org</ContactDetails>
+		<IssuingAuthority>openEuler security committee</IssuingAuthority>
+	</DocumentPublisher>
+	<DocumentTracking>
+		<Identification>
+			<ID>openEuler-SA-2021-1033</ID>
+		</Identification>
+		<Status>Final</Status>
+		<Version>1.0</Version>
+		<RevisionHistory>
+			<Revision>
+				<Number>1.0</Number>
+				<Date>2021-02-05</Date>
+				<Description>Initial</Description>
+			</Revision>
+		</RevisionHistory>
+		<InitialReleaseDate>2021-02-05</InitialReleaseDate>
+		<CurrentReleaseDate>2021-02-05</CurrentReleaseDate>
+		<Generator>
+			<Engine>openEuler SA Tool V1.0</Engine>
+			<Date>2021-02-05</Date>
+		</Generator>
```

Comment:
There is no generator field in `types`.
We try to reproduce all information from vendor.

Reply:
`DocumentTracking` doesn't contains `Generator`:
https://github.com/wjunLu/vuln-list-update/blob/412eff5c46d476550aeb1db97c270f9b3badf939/openeuler/types.go#L12-L19

Reply:
I think not all the information in cvrf files is useful for cve analysis, so some of them are not stored to the data struct.

Reply:
As I wrote earlier, we save all possible fields fro source.
`vuln-list` helps track changes, and any field can help understand changes.

Please add missing fields.
We will only use the required fields in trivy-db.